Cargo.toml

```diff
@@ -30,14 +30,14 @@ redis = ">=0.8.0"
 regex = ">=1.0"
 reqwest = ">=0.8.5"
 # Remove once a new version of rocket containing this commit is released.
-rocket = { git = "https://github.com/SergioBenitez/Rocket", rev="18a91c938eb534a410a76137fbf114333ab38643" }
-rocket_codegen = { git = "https://github.com/SergioBenitez/Rocket", rev="18a91c938eb534a410a76137fbf114333ab38643" }
-rocket_contrib = { git = "https://github.com/SergioBenitez/Rocket", rev="18a91c938eb534a410a76137fbf114333ab38643" }
-rusoto_core = ">=0.32.0"
-rusoto_credential = ">=0.11.0"
-rusoto_mock = ">=0.26.0"
-rusoto_ses = ">=0.32.0"
-rusoto_sqs = ">=0.32.0"
+rocket = { git = "https://github.com/SergioBenitez/Rocket", rev="d7f6d82fe4d35ee79c424dd4164733b22f17f80a" }
+rocket_codegen = { git = "https://github.com/SergioBenitez/Rocket", rev="d7f6d82fe4d35ee79c424dd4164733b22f17f80a" }
+rocket_contrib = { git = "https://github.com/SergioBenitez/Rocket", rev="d7f6d82fe4d35ee79c424dd4164733b22f17f80a" }
+rusoto_core = { git = "https://github.com/rusoto/rusoto/", rev="9e22eb3876a02375b3dbe14d0340f9b5c1fc55e9" }
+rusoto_credential = { git = "https://github.com/rusoto/rusoto/", rev="9e22eb3876a02375b3dbe14d0340f9b5c1fc55e9" }
+rusoto_mock = { git = "https://github.com/rusoto/rusoto/", rev="9e22eb3876a02375b3dbe14d0340f9b5c1fc55e9" }
+rusoto_ses = { git = "https://github.com/rusoto/rusoto/", rev="9e22eb3876a02375b3dbe14d0340f9b5c1fc55e9" }
+rusoto_sqs = { git = "https://github.com/rusoto/rusoto/", rev="9e22eb3876a02375b3dbe14d0340f9b5c1fc55e9" }
```

Comment:
It mildly concerns me that our dependence on git hashes instead of released versions of crates appears to be growing rather than shrinking. Are we really so desperate for the targeted features (including for the preceding one that caused the initial Rocket dependency on a commit hash) that we can't wait for them to show up in official releases?

Reply:
Yes, I don't like that either... in this case we are pointing to the 0.33.0 release commit, so it should be fine. Also I will change this to an actual version as soon as https://github.com/rusoto/rusoto/issues/1089 gets done.

We probably could have waited for them to release the rusoto version with the fixes that we need for `rusoto_mock`, but since we wanna test ASAP if the new release fixes our SES problems (#159) I just pointed to the revision for now.

The rocket revision concerns me more because that is almost a random commit, but apparently they are going to release the new version before RustConf or during RustConf, anyways, we will be able to change that soon :)